### ISSUE TITLE
Summarizing Freyja outputs

### ISSUE THREAD

**Issue by ArtPoon** [PoonLab/gromstole#80]
* from meetings last week, there are two approaches to summarizing Freyja outputs (frequency estimates for potentially as many as 2,000 defined lineages):
  1. report lineages with frequencies that exceed some threshold (e.g., 1%) and collapse all others into an "other" category
  2. sum frequencies for defined "clades" (all lineages that descend from a predefined set of ancestral lineages)
* I had proposed a compromise between (1) and (2), where lineages that are above threshold would also be annotated with their ancestral lineage in some predefined reference list.
* Presently we have a script that implements (2).  My concern is that this will require substantial manual curation on our part (*e.g.*, tracking which sublineages are to be grouped into reporting categories)
* We can look up the ancestral lineages of any given PANGO lineage from the PANGO alias JSON file (given the expanded name)
* In other words, the CSV output would contain the following fields:
  * Lineage name
  * estimated frequency
  * nearest ancestor (of interest) - if lineage is not descendant of any ancestor of interest, then this can be labelled "other"

**Comment by ArtPoon** [comment]
This is implemented
